djinni/cwrapper/wrapper_marshal.cpp

```diff
@@ -4,7 +4,7 @@
 #include <stdio.h> // for debugging
 #include "wrapper_marshal.hpp"
 
-#include "../djinni_common.hpp"
+#include "djinni/djinni_common.hpp"
```

Comment:
Specify include based in include directories rather than relative to the current dir.